components/brave_wallet_ui/common/trezor/trezor_bridge_keyring.ts

```diff
@@ -97,6 +121,34 @@ export default class TrezorBridgeKeyring extends EventEmitter {
     return ''
   }
 
+  private normalize (buf: any) {
+    return bufferToHex(buf).toString()
+  }
+
+  private prepareTransactionPayload = (path: string, txInfo: TransactionInfo, chainId: string): SignTransactionCommandPayload => {
+    const txParams = {
+      nonce: txInfo.txData.baseData.nonce,
+      gasPrice: txInfo.txData.baseData.gasPrice,
+      gasLimit: txInfo.txData.baseData.gasLimit,
+      to: txInfo.txData.baseData.to,
+      value: txInfo.txData.baseData.value,
+      data: Buffer.from(txInfo.txData.baseData.data)
+    }
+    const tx = new Transaction(txParams)
```

Comment:
I don't see why it is needed to  use `Transaction` from ethereumjs-tx.
Our `TransactionInfo` is sufficient enough to fill every fields  of `transaction` input object

Reply:
done